.changelog/v0.34.27/breaking-changes/360-update-to-go-119.md

```diff
@@ -0,0 +1,2 @@
+- Use Go 1.19 to build CometBFT, since Go 1.18 has reached end-of-life.
```

Comment:
Since there is \#385 listed below, I'm not sure including this done achieves anything (think of a user reading these). It could even be misleading.

I'd remove this one

Reply:
I'll do this in a follow-up PR.

Reply:
Oh, but isn't #385 relevant to v0.37.0, while #360 is relevant to v0.34.27?